src/structures/CommandContext.js

```diff
@@ -11,11 +9,11 @@ module.exports = class CommandContext {
   }
 
   async replyT(emoji, text, translateOptions = {}) {
-    return this.message.channel.send(`${emotes[emoji] || '🐛'} **|** ${this.message.author}, ${this.i18n(text, translateOptions)}`);
+    return this.message.channel.send(`${this.client.constants.emojis[emoji] || '🐛'} **|** ${this.message.author}, ${this.i18n(text, translateOptions)}`);
```

Comment:
acho que a pessoa passar o emoji direto seria melhor, sem colocar na instancia do cliente

```js
import { Emojis } from '@root/Constants';

ctx.replyT(Emojis.NOTIFY, "message created.");
```

Reply:
I thought in that, but then EVERY command whould have the import once every command use an emoji to respond, so wasn't better just import it once in the client, and pass it to all commands?

Reply:
acho que n pq é um valor constante e tmb cada shard teria um clone a+

Reply:
classes devem ter apenas valores mutaveis ou de configuração, n sinto que se encaixa